Assets/_Scripts/UIScripts/ScreenSwitcher.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ScreenSwitcher : MonoBehaviour {
+
+	string scale;
+	string previousScale;
+	public GameObject key1;
+
+	string sample;
+	string previousSample;
+	public GameObject sampleBtn;
+
+	public GameObject cam;
+	AudioReverbFilter reverb;
+	AudioEchoFilter echo;
+	AudioChorusFilter chorus;
+
+	float decay;
+	float delay;
+	float rate;
+
+	float previousDecay;
+	float previousDelay;
+	float previousRate;
+
+	void start(){
+		
+	}
+
+	void Update () {
+		scaleScreenMove ();
+		sampleScreenMove ();
+		effectScreenMove ();
+	}
+
+	// listens for scale being changed
+
+	void scaleScreenMove(){
```

Comment:
these functions are the same, should just be one function where you pass in the screen as an argument